Bluetooth: Controller: Fix clock accuracy calculation scanning Aux PDU

cvinayak opened the pull request:
Fix clock accuracy value used in the calculation of window
widening applied when scanning for auxiliary PDU.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

cvinayak committed:
Bluetooth: Controller: Fix clock accuracy calculation scanning Aux PDU

Fix clock accuracy value used in the calculation of window
widening applied when scanning for auxiliary PDU.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

ppryga commented on subsys/bluetooth/controller/ll_sw/ull_scan_aux.c:
```diff
@@ -277,10 +277,14 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
-	} else {
 		window_widening_us = aux_offset_us / 20000U;
+	} else {
+		window_widening_us = aux_offset_us / 2000U;
```
Maybe change the `2000` to `#define ACTIVE_CLOCK_ACURACY_RECIPROCAL 2000`?

ppryga commented on subsys/bluetooth/controller/ll_sw/ull_scan_aux.c:
```diff
@@ -277,10 +277,14 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 	if (aux_ptr->ca) {
-		window_widening_us = aux_offset_us / 2000U;
-	} else {
 		window_widening_us = aux_offset_us / 20000U;
```
Maybe change the `20000` to `#define SLEEP_CLOCK_ACURACY_RECIPROCAL 20000`?

ppryga-nordic reviewed (approved):
I've added a comment about change of number to #defines. Except that LGTM.

cvinayak committed:
Bluetooth: Controller: Use define for offset unit and SCA calculation

Use defines for offset unit and for calculation of Periodic
Advertising window widening using clock accuracy value.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>
